### ISSUE TITLE
Catch encoding exceptions when loading assignments

### ISSUE THREAD

**Issue by tdowds** [bmun/huxley#540]
When loading assignments into Huxley we often encounter errors due to Huxley not being able to read special characters (like Chinese characters, accents, tildas, etc.). We should catch these errors and tell whoever is loading assignments which assignment is throwing the exception.

**Comment by kmeht** [comment]
Yeah, I never really grokked how encoding works in Python 2. One advantage to switching to Python 3 would be that everything is unicode by default.
